packages/react-components/.eslintignore

```diff
@@ -1,3 +1,4 @@
+@types/html-to-react.d.ts
```

Comment:
Why do we need this one? The html-to-react has its own d.ts definition, is it not working?
https://www.npmjs.com/package/html-react-parser?activeTab=code
![image](https://user-images.githubusercontent.com/11863655/236062356-40b63718-396b-4643-b384-84260ac0e9bd.png)

Reply:
I'm actively working with the package owner to get that PR in; I have provided this upstream.